src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/filters/PreAuthenticatedTokenFilter.java

```diff
@@ -26,12 +31,19 @@ public class PreAuthenticatedTokenFilter implements Filter {
 
 	@Override
 	public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
-		// Extract username from a custom header added via the staff-api (this user has already been authenticated by the staff-api using keycloak)
+		// Extract username and full name from the custom headers added via the staff-api (this user has already been authenticated by the staff-api using keycloak)
 		HttpServletRequest httpRequest = (HttpServletRequest) request;
 		String username = httpRequest.getHeader("x-username");
+		String fullName = httpRequest.getHeader("fullName");

```

Comment:
I would recommend this custom header change this to x-fullname to be consistent.

Reply:
Actually, the use of "x" prefix is not officially recommended anymore based on RFC resource mentioned here: [](https://stackoverflow.com/questions/3561381/custom-http-headers-naming-conventions#:~:text=The%20reason%20is%20that%20when,the%20%22X%2D%22%20prefix.) so I named it without the prefix. However, your suggestion is right. In order to keep them consistent, it could be better to change "x-username" to  "username" only.

Reply:
Yeah, I saw the same that custom headers are no longer recommend to prefix with x-, but Phil was pretty insistent when I added that x-username field :)

Reply:
Ah I see. Alright, I will prefix it with "x" then :)